deploy/local.sh

```diff
@@ -0,0 +1 @@
+source ./deploy/single-script-install.sh --env local
```

Comment:
please remove this file. A dedicated documentation page can describe this command with other details of this localhost setup. The documentation can go to _docs/admin/localhost.md_.